### PR TITLE
Edit Read Me with Mockup & text

### LINKED ISSUE #35: Update README
- [ ] Create a UI MockUp
- [ ] Update contents to match project
- [ ] Update the link of the GitHub Actions build status badge

### PR DESCRIPTION
**Pull request by angrybunny123**
Closes #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
+**bookmark** (not your regular bookmark!) is a desktop app for tracking reading progress and book loans. It allows you to centralise all your
+reading progress into one place.
+Example usages:
+* Adding a book and its current bookmark and library due date (optional)
+* View a specific book and its details
+* Update the details of a specified book
+* Delete a specified book
+* Listing all books in the application
+## User Guide
+Interested to try out the application? Head over to the [User Guide](https://nus-cs2103-ay2021s1.github.io/tp/UserGuide.html)
```
Comment (TanLeYang) [inline README.md]: The user guide link is wrong, the correct link should be https://ay2021s1-cs2103t-f13-2.github.io/tp/UserGuide.html

File: README.md [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
+to get started!
+## Developer Guide
+bookmark is an open source project and we welcome contributions from the community. Head over to the 
+[Developer Guide](https://nus-cs2103-ay2021s1.github.io/tp/DeveloperGuide.html#product-scope) to get started!
```
Comment (TanLeYang) [inline README.md]: Same for the dev guide as well, correct link should be https://ay2021s1-cs2103t-f13-2.github.io/tp/DeveloperGuide.html

**Review by pangpuncake — APPROVED** [review]
LGTM!

**Review by pennhanlee — COMMENTED** [review]
change the UG and DG links then LGTM!!

**Review by TanLeYang — APPROVED** [review]
LGTM!

**Pull request merged**